src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/ReferralServiceTest.kt

```diff
@@ -96,6 +97,23 @@ class ReferralServiceTest @Autowired constructor(
       SampleData.sampleReferral("X123456", "Harmony Living")
     )
     sampleIntervention = sampleReferral.intervention
+
+    sampleCohortIntervention = SampleData.persistIntervention(
```

Comment:
Not really an issue (I see rest of file is using sample data). But it may be useful when persisting data in tests going forward to use the factories if we are adding new things, so we can move away from sample data eventually.